components/compliance-service/migrations/upgrades.go

```diff
@@ -0,0 +1,69 @@
+package migrations
+
+import (
+	"context"
+	"fmt"
+	"github.com/chef/automate/components/compliance-service/dao/pgdb"
+	"github.com/chef/automate/lib/cereal"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type Upgrades struct {
+	DB            *pgdb.DB
+	cerealManager *cereal.Manager
+}
+
+func New(db *pgdb.DB, cerealManager *cereal.Manager) *Upgrades {
```

Comment:
what is the need for a cereal manager here? please do not combine DB and Cereal manager flows. We can fetch the flag from DB and based on that we can write business logic to en queue of workflow. if we separate them, we can write the unit tests for DB queries and upgrade stmts.

Reply:
This change is implemented in db_integrationa and upgrade file now.